front/package.json

```diff
@@ -57,7 +57,7 @@
     "@tippyjs/react": "4.2.5",
     "ajv": "6.12.6",
     "bowser": "1.x",
-    "cl2-component-library": "0.9.21",
+    "cl2-component-library": "0.9.24",
```

Comment:
Had to publish a couple of versions as I kept finding small problems that needed to be resolved. More info (if you are interested) can be found here: https://github.com/CitizenLabDotCo/cl2-component-library/pulls?q=is%3Apr+is%3Aclosed